linchpin/provision/roles/dependencies/tasks/main.yml

```diff
@@ -10,11 +10,11 @@
   loop_control:
     loop_var: indv_prov
 
-- name: Install os-specific depdendencies:: Fedora
+- name: "Install os-specific depdendencies Fedora"
```

Comment:
Are you sure about the quotes you added? 
Another thing you have typo there depdendencies -> dependencies